packages/pricing/src/models/price-set-money-amount.ts

```diff
@@ -103,10 +108,10 @@ export default class PriceSetMoneyAmount {
     fieldName: "price_list_id",
     onDelete: "cascade",
   })
-  price_list_id: string
+  price_list_id: string | null = null
```

Comment:
todo: nullable + on delete cascade does not make much sense I believe, the only way would be that you update the item and set the price_list_id to null manually, but I believe that we never do that right otherwise you would end up with orphan data. plus, the on delete cascade means that we remove that tuple if the price list is removed. So I believe we can remove nullable wdyt?

Reply:
Do you have an example of scenario? Do we have a way to later attach a psma to a price list? For that i suppose you need first to retrieve a psma and then attach it to a price list

Reply:
[solved over chat] result: nothing to do

Reply:
All regular product variant prices fall in this category no? That is, they exist without a price list id.

Reply:
Yes exactly 💯

Reply:
what got me is that it is only doable by hand in the db to re attach to a price list